content/https/https-2/contents.lr

```diff
@@ -0,0 +1,11 @@
+_model: question
+---
+title: Can I browse normal https sites with Tor?
+---
+seo_slug: browse-normal-https-sites
+---
+description:
+Short Answer, **Yes you can browse normal HTTPS Sites using Tor.**
+
+HTTPS Connections is used for secure communication over a computer network.You can read more about HTTPS [here](https://tb-manual.torproject.org/secure-connections/)
+Tor browser have [HTTPS Everywhere](https://www.eff.org/https-everywhere) plugin which automatically switches thousands of sites from insecure "http" to secure "https".

```

Comment:
```suggestion
Tor Browser has the [HTTPS Everywhere](https://www.eff.org/https-everywhere) plugin which automatically switches thousands of sites from insecure "http" to secure "https".
```
I feel the grammar could be improved a bit :)

Reply:
yeah missed it.!🧐